src/libraries/sendtohelix.proj

```diff
@@ -97,7 +102,28 @@
   <Target Name="BuildHelixWorkItems"
           DependsOnTargets="CompressRuntimeDirectory">
 
-    <ItemGroup>
+    <ItemGroup Condition="'$(TargetOS)' == 'iOS' or '$(TargetOS)' == 'tvOS'">
+      <XHarnessAppFolderToTest Include="$([System.IO.Directory]::GetDirectories('$(TestArchiveTestsRoot)', '*.app', System.IO.SearchOption.AllDirectories))">
+        <Targets Condition="'$(TargetArchitecture)' == 'arm'">ios-device</Targets>
+        <Targets Condition="'$(TargetArchitecture)' == 'arm64'">ios-device</Targets>
+        <Targets Condition="'$(TargetArchitecture)' == 'x64'">ios-simulator-64</Targets>
+        <Targets Condition="'$(TargetArchitecture)' == 'x86'">ios-simulator-32</Targets>
+      </XHarnessAppFolderToTest>
+    </ItemGroup>
+
+    <ItemGroup Condition="'$(TargetOS)' == 'Android'">
+      <_apks Include="$(TestArchiveTestsRoot)**/*.apk" />
```

Comment:
Why introduce a separate itemgroup? We can just do this directly in `XHarnessPackageToTest`

Reply:
I was actually testing this locally and was unable to access the `%(Filename)` metadata when I had the `Include` directly in `XHarnessPackageToTest`. I debugged the binlogs yesterday for an hour and don't understand why, but it was the case. Even `%(Identity)` wasn't set..